src/constants.ts

```diff
@@ -58,9 +58,9 @@ export const WP_CLI_IMPORT_EXPORT_RESPONSE_TIMEOUT =
 	WP_CLI_IMPORT_EXPORT_RESPONSE_TIMEOUT_IN_HRS * 60 * 60 * 1000; // 6hr
 
 // SQLite
-export const SQLITE_DATABASE_INTEGRATION_VERSION = 'v2.1.17-alpha.1';
+export const SQLITE_DATABASE_INTEGRATION_VERSION = 'v2.2.2';
```

Comment:
We can use the new version v2.2.3
```suggestion
export const SQLITE_DATABASE_INTEGRATION_VERSION = 'v2.2.3;
```